Taggers/python/dumperConfigTools.py

```diff
@@ -28,8 +31,15 @@ def addCategory(pset,label,cutbased=None,subcats=0,variables=[],histograms=[],mv
     if cutbased:
         cb = cms.PSet( cut=cms.string(cutbased) )
         if( label != "" ):
+            cb.name = cms.untracked.string(label)
+           # cb.name = cms.untracked.string(systLabel)
+        if( classname != "" ): #if classname AND cutbased, then this is
```

Comment:
In fact, here, I removed the need for l.36 etc by using the default label (for the systematic dumpers, composed of `classname_systlabel`) instead of the `systLabel` alone.

So the syntax is the same regardless of dumper type, as wanted.

In order to make this work however, I needed to add additional information into the cut for each dumper, as detailed in the message for the follow up-commit below.